sky/tools/create_full_ios_framework.py

```diff
@@ -163,9 +163,34 @@ def create_framework(
   ])
 
 
+def embed_codesign_configuration(config_path, contents):
+  with open(config_path, 'w') as f:
+    f.writelines(contents)
+
+
 def zip_archive(dst):
+  ios_file_with_entitlements = ['gen_snapshot_arm64\n']
```

Comment:
nit: Instead of appending '\n' to every list entry here and below, you can instead say `'\n'.join(contents)` above in `embed_codesign_configuration`.

Reply:
True, yeah I thought about it and GN's metadata collection from `generated_file` would produce a trailing \n at the end of the file. Ended up not using `join` and align python configs with the GN produced files to have a trailing \n.

Reply:
Actually yeah it's true. It is better to do a join. I will submit a quick fix in a follow up PR.